lib/jobs/ucs-discovery.js

```diff
@@ -140,44 +140,57 @@ function UcsDiscoveryJobFactory(
                         return res.body;
                     })
                     .map(function(data) {
-                        assert.array(data.macs);
-                        assert.string(data.name);
-                        assert.string(data.path);
-
-                        var config = Object.assign({}, self.settings);
-                        obm = {
-                            config: config,
-                            service: 'ucs-obm-service'
-                        };
-                        var node = {
-                            type: 'compute',
-                            name: data.path,
-                            identifiers: data.macs,
-                            obm:[obm],
-                            relations : []
-                        };
-
-                        // Update existing node  or create a new one
-                        return waterline.nodes.needOne({identifiers:data.macs})
-                            .then(function(curNode) {
-                                return waterline.nodes.updateOne(
-                                    { id: curNode.id },
-                                    node
-                                )
-                                .then(function(data){
-                                    return data;
+                        self.createNode(data, 'compute');
+                    });
+            }else{
+                logger.warning('No rackmount servers found Found');
+                return;
+            }
+        });
+    };
+
+    /**
+     * @function createServers
+     * @description discovers all the chassis servers in the Cisco UCS
+     */
+    UcsDiscoveryJob.prototype.createServers = function (root) {
+        var self = this;
+        var username = this.settings.username;
+        var password = this.settings.password;
+        var ucs = this.settings.ucs;
+        var baseurl = this.settings.protocol+"://"+this.settings.host+ ":"+this.settings.port;
+        var url= baseurl + "/chassis?host=" + ucs+ "&user="+ username + "&password=" + password;/* jshint ignore:line */
```

Comment:
A little style suggestion. Somewhere there is space both before and after the operator `+` , sometimes only after, sometimes neither, you should at least keep it in same style.